packages/@glimmer/runtime/lib/compiled/opcodes/component.ts

```diff
@@ -755,9 +770,9 @@ APPEND_OPCODES.add(Op.SetBlocks, (vm, { op1: _state }) => {
   let state = check(vm.fetchValue(_state), CheckFinishedComponentInstance);
   let { blocks } = check(vm.stack.peek(), CheckArguments);
 
-  bindBlock('&attrs', 'attrs', state, blocks, vm);
-  bindBlock('&else', 'else', state, blocks, vm);
-  bindBlock('&default', 'main', state, blocks, vm);
+  for (let name of blocks.names) {
+    bindBlock(`&${name}`, name, state, blocks, vm);
```

Comment:
We should not be concattenating here, we should make this work the same way that `.atNames` works instead.